integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/DeleteExecution.scala

```diff
@@ -322,10 +298,117 @@ object DeleteExecution {
       resultIter
     }
 
-    if (executorErrors.failureCauses == FailureCauses.NONE) {
-       operatedRowCount = res.flatten.map(_._2._3).sum
+    (res, blockMappingVO)
+  }
+
+  // all or none : update status file, only if complete delete opeartion is successfull.
+  def checkAndUpdateStatusFiles(executorErrors: ExecutionErrors,
```

Comment:
move `executorErrors: ExecutionErrors` to next line

Reply:
ok